[E]69.cpp

```diff
@@ -0,0 +1,24 @@
+// Problem: 69. Sqrt(x) - Easy
+// Link: https://leetcode.com/problems/sqrtx/
+
+class Solution {
+public:
+    int mySqrt(int x) {
+        int l = 0, r = x;
+        while (l <= r) {
+            // cout << "l = " << l << " r = " << r << endl;
+            long mid = l + (r - l) / 2;
+            long p = mid * mid;
+            if (p > x) {
+                r = mid - 1;
+            }
```

Comment:
Nhỏ thôi. Nên để else cùng dòng với dấu ngoặc đóng của lệnh if trước đó:
if() {
} else if {
} else ...

Reply:
Và sửa cả bài 35 nữa là code đẹp đẽ :D

Reply:
Yes. Thank you.

Reply:
Done. Mời bạn check :))